torchvision/ops/deform_conv.py

```diff
@@ -0,0 +1,123 @@
+import math
+
+import torch
+from torch import nn, Tensor
+from torch.nn import init
+from torch.nn.parameter import Parameter
+from torch.nn.modules.utils import _pair
+from torch.jit.annotations import Optional, Tuple
+
+
+def deform_conv2d(input, weight, offset, bias=None, stride=(1, 1), padding=(0, 0), dilation=(1, 1)):
```

Comment:
nit: Given that `offset` can be seen as another input, I think it might make sense to move it so that the order of the arguments is `input, offset, weight, ...`
No need to change the order of the arguments in the C++ extension, just in the Python-facing function